app/components/markdown/markdown.tsx

```diff
@@ -85,7 +84,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
 
     return {
         block: {
-            alignItems: 'flex-start',
+            alignItems: 'center',
```

Comment:
What other impact does this chsnge have apart from latex?

Reply:
Anything inline or text with different sizes (which I believe we don't have).

Reply:
Inline images for example? What about headings?

Reply:
Headings are not paragraph. And now that you mentioned it, they suffer from the same alignment problem. A latex formula in a heading will be not aligned. Want me to fix?

Reply:
Inline images are affected by this, in a good way I believe, since now the image is vertically aligned with the text, as `---[image]---` instead of with the baseline, as `___[image]___`.

Reply:
Re Custom Emojis: The don't seem to be affected in my tests.